File: Python/Beginners/lanche.py
# Caio Cesar 09/11/23
# BEE 1038

codigos = {1: 4, 2: 4.5, 3: 5, 4: 2, 5: 1.5}

entrada = [int(n) for n in input().split()]

valor = codigos[entrada[0]] * entrada[1]

print(f'Total: R$ {valor:.2f}')

File: C++/Beginner/lanche.cpp
// Caio Cesar 28/11/23
// BEE 1038

#include <cstdio>

int main(){
    int codigo, quantidade;
    float valor;

    scanf("%d %d", &codigo, &quantidade);

    if(codigo == 1){
        valor = quantidade * 4.0;
    }else if(codigo == 2){
        valor = quantidade * 4.5;
    }else if(codigo == 3){
        valor = quantidade * 5.0;
    }else if(codigo == 4){
        valor = quantidade * 2.0;
    }else{
        valor = quantidade * 1.5;
    }

    printf("Total: R$ %.2f\n", valor);

    return 0;    
}